### ISSUE TITLE
log4j.properties inside generated jars

### ISSUE THREAD

**Issue by jonasabreu** [mcohen01/amazonica#173]
The `log4j.properties` inside released jars of amazonica contains the following code:

```
log4j.rootLogger=debug, A1
log4j.appender.A1=org.apache.log4j.ConsoleAppender
log4j.appender.A1.layout=org.apache.log4j.PatternLayout
log4j.appender.A1.layout.ConversionPattern=%d [%t] %-5p %c -  %m%n
log4j.logger.com.amazonaws=DEBUG
```

This is enabling debug logging for transitive dependencies os my projects, even though I use logback with slf4j. 

To work around this issue I exclude log4j from my projects and added log4j-over-slf4j, so it uses my configured logger.

Since this file is not present on this project source tree, I believe it is being included on the jars by mistake. I confirmed that the latest amazonica release (0.3.35) also has this file inside.

**Comment by mcohen01** [comment]
Sorry about that. Fixed in 0.3.36. Thank you!

**Comment by jonasabreu** [comment]
No problem! Thanks for fixing it!
